empire.go

```diff
@@ -425,6 +425,9 @@ type DeploymentsCreateOpts struct {
 	// Image is the image that's being deployed.
 	Image image.Image
 
+	// Environment is the environment where the image is being deployed
+	Environment string
```

Comment:
Hmm - yeah, I guess actually taking the environment FROM the github deployment isn't actually all that useful.  It's not like Empire runs multiple environments.  Let me think about that - doesn't seem like it'd need to be super far reaching?